bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1730,10 +1730,10 @@ spec:
                   periodSeconds: 10
                 resources:
                   limits:
-                    cpu: 500m
-                    memory: 4Gi
+                    cpu: 200m
+                    memory: 2Gi
                   requests:
-                    cpu: 500m
+                    cpu: 25m
```

Comment:
I think we should only update requests instead of limits.

Reply:
i am confused for this part. I've left some question/comments in the jira and slack, would be good to understand these data first.